sdk/python/kfp/client/set_volume_credentials.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import logging
+
+from kubernetes.client import configuration
+
+from kfp import client
+
+
+class ServiceAccountTokenVolumeCredentials(client.TokenCredentialsBase):
```

Comment:
Need to update the import reference for this one:
https://github.com/kubeflow/pipelines/blob/dfc85b0a0d7f6e59bb3b1ced03d51a24a065b471/sdk/python/kfp/client/client.py#L341